models/routes.go

```diff
@@ -1,16 +1,34 @@
 package models
 
 import (
-	"bytes"
+	bytes "bytes"
 	"encoding/json"
 )
 
 type Routes map[string]*json.RawMessage
 
-func (r *Routes) protoRoutes() *ProtoRoutes {
+func ParseRoutes(b map[string][]byte) *Routes {
+	if b == nil {
+		return nil
+	}
+
+	routes := Routes{}
+	for k, v := range b {
+		raw := json.RawMessage(v)
+		routes[k] = &raw
+	}
+
+	return &routes
+}
+
+func (r *Routes) ToProto() *ProtoRoutes {
+	if r == nil {
+		return nil
+	}
 	pr := &ProtoRoutes{
 		Routes: map[string][]byte{},
 	}
+	// pr := make(map[string][]byte)
```

Comment:
delete?

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)